We can't link against gemmlowp as it is a header only library.

Fixes:
/bin/ld: cannot find -lgemmlowp: No such file or directory

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
Upstream: https://github.com/tensorflow/tensorflow/commit/fb584589f707853d85a081c99b1b82598c2631c1
---
 tensorflow/lite/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -620,7 +620,6 @@ target_link_libraries(tensorflow-lite
     farmhash
     fft2d_fftsg2d
     flatbuffers::flatbuffers
-    gemmlowp::gemmlowp
     ml_dtypes
     ruy::ruy
     pthreadpool
     ${CMAKE_DL_LIBS}
